packages/core/src/cli/restart.ts

```diff
@@ -11,11 +10,12 @@ export async function watchFilesForRestart(
     return;
   }
 
+  const chokidar = await import('chokidar');
+
   const watcher = chokidar.watch(files, {
     ignoreInitial: true,
     // If watching fails due to read permissions, the errors will be suppressed silently.
     ignorePermissionErrors: true,
-    ignored: ['**/node_modules/**', '**/.git/**', '**/.DS_Store/**'],
```

Comment:
Why ignored is removed here, I think we need to do glob manually due to https://github.com/paulmillr/chokidar#upgrading.

Reply:
This method is only used to watch config files. Currently, there is no scenario requirement for glob and ignored.